en/includes/guides/branding/configure-ui-branding.md

```diff
@@ -672,4 +672,7 @@ Listed below are the text branding preferences you can apply to the screens in y
       <td><code>sign.up.heading</code></td>
       <td>This is the main heading of the sign-up box, providing a brief introduction to the registration page within your organization.</td>
    </tr>
-</table>
\ No newline at end of file
+</table>
+
+!!! note
+    You can only customize the provided fields. Adding custom fields to the branding preference texts is not supported.
```

Comment:
```suggestion
    Adding custom fields to the text preferences is not supported.
```

Reply:
First sentence is a little too straightforward. Let's only have the second sentence.